### PR TITLE
[fix](profile) Fix profile eject 

### PR DESCRIPTION
**Pull request by zhiqiang-hhhh**
1. We should only eject execution profiles after query has finished for a long enough time.
2. Only add execution to profile manager if enable_profile is true

### CODE REVIEW

**Review by zhiqiang-hhhh — COMMENTED** [review]
run buildall

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Review by yiguolei — DISMISSED** [review]

File: fe/fe-core/src/main/java/org/apache/doris/qe/QeProcessorImpl.java [hunk 1]
```diff
@@ -122,10 +122,11 @@ public void registerQuery(TUniqueId queryId, QueryInfo info) throws UserExceptio
         if (result != null) {
             throw new UserException("queryId " + queryId + " already exists");
         }
-
         // Should add the execution profile to profile manager, BE will report the profile to FE and FE
         // will update it in ProfileManager
-        ProfileManager.getInstance().addExecutionProfile(info.getCoord().getExecutionProfile());
+        if (info.coord.getQueryOptions().enable_profile) {
+            ProfileManager.getInstance().addExecutionProfile(info.getCoord().getExecutionProfile());
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/qe/QeProcessorImpl.java:128]: add a new method isEnableProfile in coordinator 

queryoptions may not set query profile enabled.

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Review by wangbo — APPROVED** [review]
LGTM

**Comment by zhiqiang-hhhh** [comment]
run external p0

**Comment by zhiqiang-hhhh** [comment]
run p0

**Review by yiguolei — APPROVED** [review]

**Pull request merged**